js/MVP/containers/DialogmoteContainer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Normaltekst, Sidetittel } from 'nav-frontend-typografi';
+import Lenke from 'nav-frontend-lenker';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Tilbakeknapp } from 'nav-frontend-ikonknapper';
+import { browserHistory } from 'react-router';
+import Brodsmuler from '../../components/Brodsmuler';
+import { dialogmoteBreadcrumb, statiskeURLer } from '../globals/paths';
+
+const WrappperStyled = styled.div`
+  display: flex;
+  justify-content: center;
+  background-color: #f1f1f1;
```

Comment:
kan evnt bruke @navLysGra i stedet for hardkoding av farge

Reply:
Hmm, har vi css-fargevariabler i appen vår? 😅